doc-content/enterprise-only/openshift/customimage-bc-githooks-proc.adoc

```diff
@@ -0,0 +1,44 @@
+[id='customimage-bc-githooks-proc']
+:componentimage: businesscentral
+:component: {CENTRAL}
+= Creating a custom {CENTRAL} image with Git hooks
+
+You can create a custom {CENTRAL} image that embeds Git hooks. In an authoring environment, you can use Git hooks to facilitate interaction between the internal Git repository of {CENTRAL} and an external Git repository. 
+
+[NOTE]
+====
+You can also provide Git hooks using a persistent volume or configuration map. For instructions about this method of providing Git hooks, see <<operator-deploy-central-proc>>.
+====
+
+.Procedure
+
+. Prepare the necessary Git hooks script files. For this example we use one script file names `pre-commit`. You cam use as many files as necessary. Ensure that the script files have the execute permission set.
+include::customimage-download-step.adoc[]
+. Create a `Dockerfile` that defines a custom image based on the base image. The file must copy the script files into the `/opt/kie/data/git/hooks/` directory. The following example shows the content of the `Dockerfile` file:
++
+[subs="attributes,verbatim,macros,quotes"]
+----
+FROM registry.redhat.io/{PRODUCT_INIT}-7/{PRODUCT_INIT}-businesscentral-rhel8:7.7.0
+COPY pre-commit /opt/kie/data/git/hooks/
+----
++
+include::customimage-build-step.adoc[]
+include::customimage-verify-partstep.adoc[]
+In the shell prompt for the image, list the contents of the `/opt/kie/data/git/hooks/` directory:
++
+[subs="attributes,verbatim,macros,quotes"]
+----
+[jboss@c2fab36b778e ~]$ ls /opt/kie/data/git/hooks/
+pre-commit
+[jboss@c2fab36b778e ~]$ exit
+----
++
+include::customimage-push-step.adoc[]
+
+.Next steps
```

Comment:
I don't think this is the intended use of Next Steps. I would make this a final step. To deploy... This occurs in a few places.